File: 81-search-in-rotated-sorted-array-ii/search-in-rotated-sorted-array-ii.py
class Solution(object):
    def search(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: bool
        """

        if target not in nums:
            return False ;
        else:
            return True ;    
        # for i in nums:
        #     if target == i:
        #         return True ;
        #         break ;
        #     if target
        

File: 81-search-in-rotated-sorted-array-ii/search-in-rotated-sorted-array-ii.java
class Solution {
    public boolean search(int[] nums, int target) {
        
        int n = nums.length ;
        int count = 0 ;
        for (int i = 0 ; i < n;i++){
            if (nums[i] == target){
                count += 1 ;
                break;
            }
            else{
                count = 0;
            }
            
        }

        if (count == 1){
            return true ;
        }
        else{
            return false;
        }
    }
}